docs/docs/integrations/chat/naver.ipynb

```diff
@@ -0,0 +1,408 @@
+{
+ "cells": [
+  {
+   "cell_type": "raw",
+   "id": "afaf8039",
+   "metadata": {},
+   "source": [
+    "---\n",
+    "sidebar_label: Naver\n",
+    "---"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "e49f1e0d",
+   "metadata": {},
+   "source": [
+    "# ChatClovaX\n",
+    "\n",
+    "This notebook provides a quick overview for getting started with Naver’s HyperCLOVA X [chat models](https://python.langchain.com/v0.2/docs/concepts/#chat-models) via CLOVA Studio. For detailed documentation of all ChatClovaX features and configurations head to the [API reference](https://api.python.langchain.com/en/latest/community/chat_models/langchain_community.chat_models.naver.ChatClovaX.html).\n",
```

Comment:
please link to recent docs at /docs, don't link to /v0.2 docs

Reply:
also api reference should start with python.langchain.com/api_reference, not api.python

Reply:
Thanks for the comments including below ones. Now self reviewing docs and codes finding what you mentioned are the most part to be modified. 

Will fix them soon, but I have a quick question to be sure as you mentioned "a template that is out of date": 
when writing docs, now I'm relying on those in [langchain_cli.integration_template](https://github.com/langchain-ai/langchain/tree/master/libs/cli/langchain_cli/integration_template), and some of the most recent updates in langchain_community like [SambaStudio](https://python.langchain.com/docs/integrations/chat/sambastudio/), look they have some tiny conflicts with your comments. Is there any up-to-date reference template to look up?